packages/test-harness/src/config/init.lua

```diff
@@ -0,0 +1,10 @@
+local temp_dir = os.getenv('TEMP_DIR')
+local repo_root = os.getenv('CURSORLESS_REPO_ROOT')
+
+vim.cmd('source ' .. temp_dir .. '/BufOnly.vim/plugin/BufOnly.vim')
+
+vim.o.runtimepath = vim.o.runtimepath .. ',' .. temp_dir .. '/talon.nvim'
```

Comment:
It's not a builtin lua function, but rather a neovim option method. The append function just basically abstracts what you did to make it cleaner, and allows appending to various option types. using :append() instead of .append() is just lua syntactic sugar for functions that pass self to the function.

You can see a description here https://github.com/neovim/neovim/blob/20f22f75ee629ae2db4cd99e730fa0af26553177/runtime/lua/vim/_options.lua#L769 and the implementation elsewhere in that file.

you can see a usage example here https://github.com/neovim/neovim/blob/20f22f75ee629ae2db4cd99e730fa0af26553177/contrib/minimal.lua#L11

otherwise just search github for 'language:lua runtimepath:append' to see how others use it.

Not super important, but will make it cleaner to read anyway.

Reply:
done